at_functional_test/test/all_verbs_test.dart

```diff
@@ -100,6 +99,37 @@ void main() async {
     print('config verb response $response');
     expect(response, contains('data:null'));
   });
+//FOR THESE TESTS TO WORK/PASS SET testingMode TO TRUE THROUGH ENV VARIABLES
+//THE FOLLOWING TESTS ONLY WORK WHEN IN TESTING MODE
+//   test('config verb test set/reset operation', () async {
+//     await socket_writer(
+//         socketFirstAtsign!, 'config:set:commitLogCompactionFrequencyMins=4');
+//     var response = await read();
+//     print('config verb response $response');
+//     expect(response, contains('data:ok'));
+//
+//     await socket_writer(
+//         socketFirstAtsign!, 'config:reset:commitLogCompactionFrequencyMins');
+//     await Future.delayed(Duration(seconds: 2));
+//     response = await read();
+//     print('config verb response $response');
+//     expect(response, contains('data:ok'));
+//   });
+//
+//   test('config verb test set/print', () async {
+//     await socket_writer(
+//         socketFirstAtsign!, 'config:set:maxNotificationRetries=25');
+//     var response = await read();
+//     print('config verb response $response');
+//     expect(response, contains('data:ok'));
+//
+//     await socket_writer(
+//         socketFirstAtsign!, 'config:print:maxNotificationRetries');
+//     await Future.delayed(Duration(seconds: 2));
+//     response = await read();
+//     print('config verb response $response');
+//     expect(response, contains('data:25'));
+//   });
```

Comment:
Please uncomment these tests. When people are running tests they need to set the env for their container. Perhaps add the documentation as to how to do this. Perhaps you can assert that testingMode is true on the server somehow (?) and if it's not then show a message describing what you need to do to the container you are running tests against.

Reply:
I have enabled the tests and am asserting that testingMode is true.